pages/index.js

```diff
@@ -1,7 +1,7 @@
 import Head from "next/head";
 import styled from "styled-components";
 import { Inter } from "next/font/google";
-import KidsList from "@/components/KidsList/KidsList";
+import KidsList from "@/components/KidsList/index";
```

Comment:
You can simplify it to just this
```suggestion
import KidsList from "@/components/KidsList";
```
Imports automagically look for an `index` file in the path.